### ISSUE TITLE
[BUG] Hydric System level from 2015-06-19

### ISSUE THREAD

**Issue by rafaell-lycan** [rafaell-lycan/sabesp-mananciais-api#31]
Now, sabesp have a different information about Cantareira, like "Índice 1: 20,0 %Índice 2: 15,4 %Índice 3: -9,3 %" on key  "volume_armazenado".

``` javascript
{
  "name": "Cantareira",
  "data": {
    "volume_armazenado": "Índice 1: 20,0 %Índice 2: 15,4 %Índice 3: -9,3 %",
    "pluviometria_do_dia": "0,0 mm",
    "pluviometria_acumulada_no_mes": "39,5 mm",
    "media_historica_do_mes": "58,5 mm"
  }
}
```

We need to prevent this behavior when filter the response.

**Comment by wbruno** [comment]
Or not.. its a list.

**Comment by rafaell-lycan** [comment]
Yeah, but we need to format the output right? Because this kind of response as string doesn't make sense. :ghost:

**Comment by wbruno** [comment]
Fix Hydric System Level formater #35
